tests/cases/fourslash/codeFixSpellingPrivatePropertyName.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path='fourslash.ts' />
+
+////class A {
+////   #foo: number;
```

Comment:
What happens if there is a property named `"#foo"` instead?

Reply:
@ajafff Good point, I didn't test that case.